docs/content/integrations/dbt.mdx

```diff
@@ -1,380 +1,159 @@
-
-- `warn_error`: will raise an error for issues that dbt would normally just warn on
-- `target_path`: the dbt target path, if you set it to something other than the default
-- `dbt_executable`: the name of the specific dbt executable you're using, if it's not just `dbt`
-
-### CLI Examples
-
-Below are some examples of using `dagster_dbt` library to run a `dbt` project via the CLI.
-
-For these examples, we make heavy use of the `dbt_cli_resource`. This resource has methods corresponding to most dbt commands (such as `run`, `seed`, `test`, `ls`, etc.). The methods behave essentially identically to each other, so these examples will focus on `dbt run` for simplicity.
-
-The `dagster_dbt` library provides ops that simplify cases where you simply wish to run a single one of these commands (with no special arguments) and return the result. This saves you from having to write out the boilerplate associated with creating an op from scratch. However, it's fairly common to have more complex needs, and in those cases you should create your own custom op that invokes the resource directly. The below examples will mix and match these two approaches where appropriate.
-
-Note that you can pass in any keyword to these functions that you wish, and they will get added as flags to the underlying dbt command (e.g. `my_flag_name = 'foo'` will get converted to `--my-flag-name foo`). If there is a dbt option that you would like to set, but is not reflected in the function signature, this is how you would do so.
-
-#### Using dbt_cli_resource to run your entire dbt project
-
-One common way to use this integration is to have the a step in your job run all of the models in a dbt project. For this case, the easiest method is to configure the resource so it knows where your dbt project is, and import the `dbt_run_op` to use in your job.
-
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_run endbefore=end_marker_dbt_cli_run dedent=4
-from dagster_dbt import dbt_cli_resource, dbt_run_op
-
-from dagster import job
-
-my_dbt_resource = dbt_cli_resource.configured(
-    {"project_dir": "path/to/dbt/project"}
+dbt_assets = load_assets_from_dbt_project(
+    project_dir="path/to/dbt/project"
 )
-
-@job(resource_defs={"dbt": my_dbt_resource})
-def my_dbt_job():
-    dbt_run_op()
 ```
 
-#### Using dbt_cli_resource to run a specific set of models
-
-Sometimes, you just want to run a select set of models in your dbt project, rather than the entire thing. The below examples show two ways of doing this, depending on your use case.
+The `load_assets_from_dbt_project` function will compile your dbt project, parse the metadata that dbt provides, and generate a set of software-defined assets that reflect the models in the project. These assets will all share the same underlying operation, which will invoke dbt to run the models represented by the loaded assets.
```

Comment:
Suggesting an alternative formatting approach to break this up and make it more readable:

```suggestion
The `load_assets_from_dbt_project` function:

1. Compiles your dbt project,
2. Parses the metadata that dbt provides, and
3. Generates a set of software-defined assets that reflect the models in the project. These assets will share the same underlying operation, which will invoke dbt to run the models represented by the loaded assets.
```